### PR TITLE
docs: create workflow on dashboard

### PR DESCRIPTION
**Pull request by STRRL**
Signed-off-by: STRRL <str_ruiling@outlook.com>

### CODE REVIEW

File: i18n/zh/docusaurus-plugin-content-docs/version-2.1.0/create-chaos-mesh-workflow.md [hunk 1]
```diff
@@ -24,6 +24,36 @@ Chaos Mesh Workflow 在设计时一定程度上参考了 Argo Workflow。如果
...
+
+### 第 3 步：配置工作流节点
+
+:::note 注意
+
+Chaos Dashboard 会自动创建一个名为 `entry`的串行(`Serial`)节点，作为该工作流的入口。
+
+:::
+
+![Choose Task Type](./img/choose-task-type.png)
```
Comment (g1eny0ung) [inline i18n/zh/docusaurus-plugin-content-docs/version-2.1.0/create-chaos-mesh-workflow.md:45]: 这个文件看改动应该是命名错了。

File: i18n/zh/docusaurus-plugin-content-docs/version-2.1.0/create-chaos-mesh-workflow.md [hunk 2]
```diff
@@ -24,6 +24,36 @@ Chaos Mesh Workflow 在设计时一定程度上参考了 Argo Workflow。如果
...
+
+例如，像创建一个一个普通的混沌一样，设置一个名为 `kill-nginx` 的 `PodChaos`。
```
Comment (g1eny0ung) [inline i18n/zh/docusaurus-plugin-content-docs/version-2.1.0/create-chaos-mesh-workflow.md:47]: > 创建一个普通的混沌实验

这样呢？

**Review by g1eny0ung — APPROVED** [review]
Rest LGTM.

File: i18n/zh/docusaurus-plugin-content-docs/current/create-chaos-mesh-workflow.md [hunk 3]
```diff
@@ -24,10 +24,39 @@ Chaos Mesh Workflow 在设计时一定程度上参考了 Argo Workflow。如果
...
+
+### 第 2 步：设置基本工作流信息
+
+![Workflow Info](./img/workflow-info.png)
+
+### 第 3 步：配置工作流节点
+
+:::note 注意
+
+Chaos Dashboard 会自动创建一个名为 `entry`的串行(`Serial`)节点，作为该工作流的入口。
```
Comment (en-jin19) [inline i18n/zh/docusaurus-plugin-content-docs/current/create-chaos-mesh-workflow.md:41]:
```suggestion
## 通过 Chaos Dashboard 创建 Workflow

### 第 1 步：打开 Chaos Dashboard

点击**新的工作流** (Workflow)。

![New Workflow](./img/new-workflow.png)

### 第 2 步：设置基本工作流信息

![Workflow Info](./img/workflow-info.png)

### 第 3 步：配置工作流节点

:::note 注意

Chaos Dashboard 会自动创建一个名为 `entry` 的串行 (`Serial`) 节点，作为该工作流的入口。
```

File: i18n/zh/docusaurus-plugin-content-docs/current/create-chaos-mesh-workflow.md [hunk 4]
```diff
@@ -24,10 +24,39 @@ Chaos Mesh Workflow 在设计时一定程度上参考了 Argo Workflow。如果
...
+
+:::
+
+![Choose Task Type](./img/choose-task-type.png)
+
+例如，像创建一个一个普通的混沌一样，设置一个名为 `kill-nginx` 的 `PodChaos`。
```
Comment (en-jin19) [inline i18n/zh/docusaurus-plugin-content-docs/current/create-chaos-mesh-workflow.md:47]:
```suggestion
配置方法与创建普通的混沌实验相同。比如，你可以设置一个名为 `kill-nginx` 的 `PodChaos`。
```

File: i18n/zh/docusaurus-plugin-content-docs/current/create-chaos-mesh-workflow.md [hunk 5]
```diff
@@ -24,10 +24,39 @@ Chaos Mesh Workflow 在设计时一定程度上参考了 Argo Workflow。如果
...
+
+![Create podkill in Workflow](./img/create-podkill-in-workflow.png)
+
+### 第 4 步：提交工作流
+
+你可以在“预览”查看工作流定义，点击“提交工作流”按钮创建工作流。
```
Comment (en-jin19) [inline i18n/zh/docusaurus-plugin-content-docs/current/create-chaos-mesh-workflow.md:53]:
```suggestion
你可以先在“预览”查看工作流定义之后，通过点击“提交工作流”按钮创建工作流。
```

File: i18n/zh/docusaurus-plugin-content-docs/version-2.1.1/create-chaos-mesh-workflow.md [hunk 6]
```diff
@@ -24,6 +24,36 @@ Chaos Mesh Workflow 在设计时一定程度上参考了 Argo Workflow。如果
...
+例如，像创建一个一个普通的混沌一样，设置一个名为 `kill-nginx` 的 `PodChaos`。
+
+![Create podkill in Workflow](./img/create-podkill-in-workflow.png)
+
+### 第 4 步：提交工作流
+
+你可以在“预览”查看工作流定义，点击“提交工作流”按钮创建工作流。
+
+![Submit Workflow](./img/submit-workflow.png)
+
```
Comment (en-jin19) [inline i18n/zh/docusaurus-plugin-content-docs/version-2.1.1/create-chaos-mesh-workflow.md:56]:
```suggestion
## 通过 Chaos Dashboard 创建工作流

### 第 1 步：打开 Chaos Dashboard

点击**新的工作流** (Workflow)。

![New Workflow](./img/new-workflow.png)

### 第 2 步：设置工作流基本信息

![Workflow Info](./img/workflow-info.png)

### 第 3 步：配置工作流节点

:::note 注意

Chaos Dashboard 会自动创建一个名为 `entry` 的串行 (`Serial`) 节点，作为该工作流的入口。

:::

![Choose Task Type](./img/choose-task-type.png)

配置方法与创建普通的混沌实验相同。比如，你可以设置一个名为 `kill-nginx` 的 `PodChaos`。

![Create podkill in Workflow](./img/create-podkill-in-workflow.png)

### 第 4 步：提交工作流

你可以先在“预览”查看工作流定义之后，通过点击“提交 Workflow”按钮创建工作流。

![Submit Workflow](./img/submit-workflow.png)

```

**Comment by en-jin19** [comment]
@STRRL 为统一同一文档中的表达，可以请把中文文档中的 Workflow 统一替换为"工作流“吗？由于本文档中有图片，所以铜罐驿为图片中的”工作流“应该会更好些。

File: docs/create-chaos-mesh-workflow.md [hunk 7]
```diff
@@ -24,6 +24,36 @@ The design of Chaos Mesh Workflow is, to some extent, inspired by Argo Workflows
...
+For example, setup a podkill `PodChaos` called `kill-nginx`, as same as creating a normal chaos experiment.
+
+![Create podkill in Workflow](./img/create-podkill-in-workflow.png)
+
+### Step 4. Submit Workflow
+
+You could check workflow definition with "Preview", and click "SUBMIT WORKFLOW" to create the workflow.
+
+![Submit Workflow](./img/submit-workflow.png)
+
```
Comment (en-jin19) [inline docs/create-chaos-mesh-workflow.md:56]:
```suggestion
## Create a workflow using Chaos Dashboard

### Step 1. Open Chaos Dashboard

Click **NEW WORKFLOW**.

![New Workflow](./img/new-workflow.png)

### Step 2. Setup basic information of the workflow

![Workflow Info](./img/workflow-info.png)

### Step 3. Configure the nodes of the workflow

:::note

Chaos Dashboard automatically creates a serial node named `entry` as the entry point for this workflow.

:::

![Choose Task Type](./img/choose-task-type.png)

The configuration method is the same as creating a normal chaos experiment. For example, you can set up a pod-kill type of `PodChaos` named `kill-nginx`.

![Create podkill in Workflow](./img/create-podkill-in-workflow.png)

### Step 4. Submit the workflow

You can check workflow definition through "Preview", and then click the "SUBMIT WORKFLOW" button to create the workflow.

![Submit Workflow](./img/submit-workflow.png)

```

File: versioned_docs/version-2.1.1/create-chaos-mesh-workflow.md [hunk 8]
```diff
@@ -24,6 +24,36 @@ The design of Chaos Mesh Workflow is, to some extent, inspired by Argo Workflows
...
+For example, setup a podkill `PodChaos` called `kill-nginx`, as same as creating a normal chaos experiment.
+
+![Create podkill in Workflow](./img/create-podkill-in-workflow.png)
+
+### Step 4. Submit Workflow
+
+You could check workflow definition with "Preview", and click "SUBMIT WORKFLOW" to create the workflow.
+
+![Submit Workflow](./img/submit-workflow.png)
+
```
Comment (en-jin19) [inline versioned_docs/version-2.1.1/create-chaos-mesh-workflow.md:56]:
```suggestion
## Create a workflow using Chaos Dashboard

### Step 1. Open Chaos Dashboard

Click **NEW WORKFLOW**.

![New Workflow](./img/new-workflow.png)

### Step 2. Setup basic information of the workflow

![Workflow Info](./img/workflow-info.png)

### Step 3. Configure the nodes of the workflow

:::note

Chaos Dashboard automatically creates a serial node named `entry` as the entry point for this workflow.

:::

![Choose Task Type](./img/choose-task-type.png)

The configuration method is the same as creating a normal chaos experiment. For example, you can set up a pod-kill type of `PodChaos` named `kill-nginx`.

![Create podkill in Workflow](./img/create-podkill-in-workflow.png)

### Step 4. Submit the workflow

You can check workflow definition through "Preview", and then click the "SUBMIT WORKFLOW" button to create the workflow.

![Submit Workflow](./img/submit-workflow.png)

```

**Review by en-jin19 — CHANGES_REQUESTED** [review]
I have finished the language review, could you please take a look? thanks!

File: i18n/zh/docusaurus-plugin-content-docs/version-2.1.0/create-chaos-mesh-workflow.md (hunk 1 above)
Comment (STRRL) [reply i18n/zh/docusaurus-plugin-content-docs/version-2.1.0/create-chaos-mesh-workflow.md]: 是指文件名错了吗？ 

`choose-task-type.png` 看上去没有问题呀；

File: i18n/zh/docusaurus-plugin-content-docs/version-2.1.0/create-chaos-mesh-workflow.md (hunk 2 above)
Comment (STRRL) [reply i18n/zh/docusaurus-plugin-content-docs/version-2.1.0/create-chaos-mesh-workflow.md]: @en-jin19 老师在下面提出更改了，俺直接应用 ta 的建议了

**Comment by STRRL** [comment]
> @STRRL 为统一同一文档中的表达，可以请把中文文档中的 Workflow 统一替换为"工作流“吗？由于本文档中有图片，所以铜罐驿为图片中的”工作流“应该会更好些。

好的！ 搜了一下使用 “Workflow” 的地方还是比较多的，我将提个新 PR 里作出更改 :D

**Comment by STRRL** [comment]
PTAL

/cc @en-jin19 @g1eny0ung

**Comment by STRRL** [comment]
I closed this document with misoperation, I would update this PR recently.

**Comment by cwen0** [comment]
@STRRL Any updates?

**Comment by STRRL** [comment]
> @STRRL Any updates?

I would update this PR in this week.

**Comment by STRRL** [comment]
PTAL 
/cc @g1eny0ung @cwen0 @en-jin19

**Comment by STRRL** [comment]
Hi @g1eny0ung, @en-jin19 noticed me that there are some unexpected changes like `docs/simulate-dns-chaos-on-kubernetes.md`, it turns tab from "4 spaces" to "3 spaces". 

@en-jin19  suggests to me that we would better use 4 spaces.

And I found that these changes might be caused by `husky` and `prettier`, could you help us to change the formatting rules? Thanks very much! @g1eny0ung

**Comment by g1eny0ung** [comment]
> Hi @g1eny0ung, @en-jin19 noticed me that there are some unexpected changes like `docs/simulate-dns-chaos-on-kubernetes.md`, it turns tab from "4 spaces" to "3 spaces".
> 
> @en-jin19 suggests to me that we would better use 4 spaces.
> 
> And I found that these changes might be caused by `husky` and `prettier`, could you help us to change the formatting rules? Thanks very much! @g1eny0ung

@STRRL Do you mean this? This is normal behavior. This aligns the code with the list text.

image

File: i18n/zh/docusaurus-plugin-content-docs/current/create-chaos-mesh-workflow.md [hunk 9]
```diff
@@ -24,6 +24,38 @@ Chaos Mesh Workflow 在设计时一定程度上参考了 Argo Workflow。如果
...
+配置方法与创建普通的混沌实验相同。比如，你可以设置一个名为 `kill-nginx` 的 pod-kill 类型的 `PodChaos` 故障。
+
+![Create podkill in Workflow](./img/create-podkill-in-workflow.png)
+
+### 第 4 步：提交工作流
+
+你可以先在“预览”查看工作流定义之后，通过点击“提交 Workflow”按钮创建工作流。
+
+![Submit Workflow](./img/submit-workflow.png)
+
```
Comment (en-jin19) [inline i18n/zh/docusaurus-plugin-content-docs/current/create-chaos-mesh-workflow.md:58]:
```suggestion
## 通过 Chaos Dashboard 创建 Workflow

### 第 1 步：打开 Chaos Dashboard

点击**新的工作流**。

![New Workflow](./img/new-workflow.png)

### 第 2 步：设置工作流基本信息

![Workflow Info](./img/workflow-info.png)

### 第 3 步：配置工作流节点

1. 选择**工作流入口**

:::note 注意

如果未设置工作流入口，Chaos Dashboard 会自动创建一个串行 (Serial) 节点，作为该工作流的入口，并将其命名为 “entry”。

:::

![Choose Task Type](./img/choose-task-type.png)

2. 填写实验信息

配置方法与创建普通的混沌实验相同。比如，你可以设置一个 POD KILL 类型的 PodChaos 故障，并将其命名为 “kill-nginx”。 

![Create podkill in Workflow](./img/create-podkill-in-workflow.png)

### 第 4 步：提交工作流

你可以先在**预览**查看工作流定义。确认无误后，点击**提交工作流**按钮创建工作流。

![Submit Workflow](./img/submit-workflow.png)

```

**Comment by STRRL** [comment]
> > Hi @g1eny0ung, @en-jin19 noticed me that there are some unexpected changes like `docs/simulate-dns-chaos-on-kubernetes.md`, it turns tab from "4 spaces" to "3 spaces".
> > @en-jin19 suggests to me that we would better use 4 spaces.
> > And I found that these changes might be caused by `husky` and `prettier`, could you help us to change the formatting rules? Thanks very much! @g1eny0ung
> 
> @STRRL Do you mean this? This is normal behavior. This aligns the code with the list text.
> 
> image

Yes. So it is an expected change?  FYI @en-jin19 


And I found that current documents in `master` branch do not follow this "aligns the code with the list text" rule.

https://github.com/chaos-mesh/website/blob/e66b8deea3a344b18446d4d68851f73c8171df86/docs/simulate-dns-chaos-on-kubernetes.md?plain=1#L31-L36

Should we add another CI checker for this?

**Comment by g1eny0ung** [comment]
> Should we add another CI checker for this?

Probably not, because it doesn't affect the actual display.

File: i18n/zh/docusaurus-plugin-content-docs/current/create-chaos-mesh-workflow.md (hunk 9 above)
Comment (en-jin19) [reply i18n/zh/docusaurus-plugin-content-docs/current/create-chaos-mesh-workflow.md:58]: @STRRL 

1. 目前中文中第 3 步中 1 的 /img/choose-task-type.png （修改内容中的 49 行）图片与英文的对应图片截图内容不一样，可以请老师看下是否需要重新上传呢？如需上传，可以请根据上文内容，截图时把鼠标挪到“串流”部分吗？这样能提示在本示例中用的是“串流”节点。
2. 在中文中第 3 步 2 的 /create-podkill-in-workflow.png （修改内容中的 57 行）图片中，最上面的“工作流入口”选的是“单一”，而提供的“预览”（img/submit-workflow.png）图片中，该部分显示的是“Serial”。可以请老师把 /create-podkill-in-workflow.png 图片替换为【“单一”改成“串流”】的图片吗？这样应该能保证整个示例中的内容都是统一的。

Comment (STRRL) [reply i18n/zh/docusaurus-plugin-content-docs/current/create-chaos-mesh-workflow.md:58]: > 

已经更新图片~  

另外示例中使用的不是 “串行”  而是 “单一” 的 PodChaos ；  “串行”节点是自动创建的；

Comment (STRRL) [reply i18n/zh/docusaurus-plugin-content-docs/current/create-chaos-mesh-workflow.md]: 另外关于 suggestion 中的 "如果未设置 Choose task type，Chaos Dashboard 会自动创建一个命名为 “entry” 串行节点，作其用为该工作流的入口。" 是准确的；

Chaos Dashbaord 总会自动创建一个命名为 “entry” 串行节点；

File: i18n/zh/docusaurus-plugin-content-docs/current/create-chaos-mesh-workflow.md [hunk 10]
```diff
@@ -24,6 +24,44 @@ Chaos Mesh Workflow 在设计时一定程度上参考了 Argo Workflow。如果
...
+### 第 2 步：设置工作流基本信息
+
+![Workflow Info](./img/workflow-info.png)
+
+### 第 3 步：配置工作流节点
+
+1. 选择**Choose task type**
+
+根据需求选择**Choose task type**。在本示例中，工作流入口选为串行 (Serial) 节点，并将其命名为 “entry”。
```
Comment (en-jin19) [inline i18n/zh/docusaurus-plugin-content-docs/current/create-chaos-mesh-workflow.md:43]:
```suggestion
1. 根据需求选择**选择任务类型**

在本示例中，工作流入口选为串行节点，并将其命名为 “entry”。
```

File: i18n/zh/docusaurus-plugin-content-docs/current/create-chaos-mesh-workflow.md [hunk 11]
```diff
@@ -24,6 +24,44 @@ Chaos Mesh Workflow 在设计时一定程度上参考了 Argo Workflow。如果
...
+
+### 第 2 步：设置工作流基本信息
+
+![Workflow Info](./img/workflow-info.png)
+
+### 第 3 步：配置工作流节点
+
+1. 根据需求选择**选择任务类型**
+
+在本示例中，工作流入口选为串行 (Serial) 节点，并将其命名为 “entry”。
```
Comment (en-jin19) [inline i18n/zh/docusaurus-plugin-content-docs/current/create-chaos-mesh-workflow.md:43]:
```suggestion
在本示例中选择的任务类型为”单一“。
```

File: i18n/zh/docusaurus-plugin-content-docs/version-2.1.2/create-chaos-mesh-workflow.md [hunk 12]
```diff
@@ -24,6 +24,38 @@ Chaos Mesh Workflow 在设计时一定程度上参考了 Argo Workflow。如果
...
+
+配置方法与创建普通的混沌实验相同。比如，你可以设置一个名为 `kill-nginx` 的 pod-kill 类型的 `PodChaos` 故障。
+
+![Create podkill in Workflow](./img/create-podkill-in-workflow.png)
+
+### 第 4 步：提交工作流
+
+你可以先在“预览”查看工作流定义之后，通过点击“提交 Workflow”按钮创建工作流。
+
+![Submit Workflow](./img/submit-workflow.png)
```
Comment (en-jin19) [inline i18n/zh/docusaurus-plugin-content-docs/version-2.1.2/create-chaos-mesh-workflow.md:57]:
```suggestion
## 通过 Chaos Dashboard 创建 Workflow

### 第 1 步：打开 Chaos Dashboard

点击**新的工作流**。

![New Workflow](./img/new-workflow.png)

### 第 2 步：设置工作流基本信息

![Workflow Info](./img/workflow-info.png)

### 第 3 步：配置工作流节点

1. 根据需求选择**选择任务类型**

在本示例中选择的任务类型为”单一“。

:::note 注意

Chaos Dashboard 会自动创建一个命名为 “entry” 串行节点，作其用为该工作流的入口。

:::

![Choose Task Type](./img/choose-task-type.png)

2. 填写实验信息

配置方法与创建普通的混沌实验相同。比如，你可以设置一个 POD KILL 类型的 PodChaos 故障，并将其命名为 “kill-nginx”。

![Create podkill in Workflow](./img/create-podkill-in-workflow.png)

### 第 4 步：提交工作流

你可以先在**预览**查看工作流定义。确认无误后，点击**提交工作流**按钮创建工作流。

![Submit Workflow](./img/submit-workflow.png)
```

File: docs/create-chaos-mesh-workflow.md [hunk 13]
```diff
@@ -24,6 +24,38 @@ The design of Chaos Mesh Workflow is, to some extent, inspired by Argo Workflows
...
+The configuration method is the same as creating a normal chaos experiment. For example, you can set up a pod-kill type of `PodChaos` named `kill-nginx`.
+
+![Create podkill in Workflow](./img/create-podkill-in-workflow.png)
+
+### Step 4. Submit the workflow
+
+You can check workflow definition through "Preview", and then click the "SUBMIT WORKFLOW" button to create the workflow.
+
+![Submit Workflow](./img/submit-workflow.png)
```
Comment (en-jin19) [inline docs/create-chaos-mesh-workflow.md:57]:
```suggestion
### Step 3. Configure the nodes of the workflow

1. Select an option under **Choose task type** according to your needs.

In this example, the "Single” type is selected as the task type.

:::note

Chaos Dashboard automatically creates a serial node named "entry" as the entry point for this workflow.

:::

![Choose Task Type](./img/choose-task-type.png)

2. Fill out the experiment information.

The configuration method is the same as creating a normal chaos experiment. For example, you can set up a "POD KILL" type of "PodChaos" named `kill-nginx`.

![Create podkill in Workflow](./img/create-podkill-in-workflow.png)

### Step 4. Submit the workflow

You can check workflow definition through **Preview**, and then click the **SUBMIT WORKFLOW** to create the workflow.

![Submit Workflow](./img/submit-workflow.png)
```

File: versioned_docs/version-2.1.2/create-chaos-mesh-workflow.md [hunk 14]
```diff
@@ -24,6 +24,38 @@ The design of Chaos Mesh Workflow is, to some extent, inspired by Argo Workflows
...
+
+The configuration method is the same as creating a normal chaos experiment. For example, you can set up a pod-kill type of `PodChaos` named `kill-nginx`.
+
+![Create podkill in Workflow](./img/create-podkill-in-workflow.png)
+
+### Step 4. Submit the workflow
+
+You can check workflow definition through "Preview", and then click the "SUBMIT WORKFLOW" button to create the workflow.
+
+![Submit Workflow](./img/submit-workflow.png)
```
Comment (en-jin19) [inline versioned_docs/version-2.1.2/create-chaos-mesh-workflow.md:57]:
```suggestion
### Step 3. Configure the nodes of the workflow

1. Select an option under **Choose task type** according to your needs.

In this example, the "Single” type is selected as the task type.

    :::note

    Chaos Dashboard automatically creates a serial node named "entry" as the entry point for this workflow.

:::

    ![Choose Task Type](./img/choose-task-type.png)

2. Fill out the experiment information.

    The configuration method is the same as creating a normal chaos experiment. For example, you can set up a "POD KILL" type of "PodChaos" named `kill-nginx`.

    ![Create podkill in Workflow](./img/create-podkill-in-workflow.png)

### Step 4. Submit the workflow

You can check workflow definition through **Preview**, and then click the **SUBMIT WORKFLOW** to create the workflow.

![Submit Workflow](./img/submit-workflow.png)
```

File: i18n/zh/docusaurus-plugin-content-docs/current/create-chaos-mesh-workflow.md [hunk 15]
```diff
@@ -24,6 +24,44 @@ Chaos Mesh Workflow 在设计时一定程度上参考了 Argo Workflow。如果
...
+
+:::
+
+![Choose Task Type](./img/choose-task-type.png)
+
+2. 填写实验信息
+
+配置方法与创建普通的混沌实验相同。比如，你可以设置一个 POD KILL 类型的 PodChaos 故障，并将其命名为 “kill-nginx”。
+
+![Create podkill in Workflow](./img/create-podkill-in-workflow.png)
```
Comment (en-jin19) [inline i18n/zh/docusaurus-plugin-content-docs/current/create-chaos-mesh-workflow.md:57]:
```suggestion
### 第 3 步：配置工作流节点

1. 根据需求选择**选择任务类型**

    在本示例中选择的任务类型为”单一“。

    :::note 注意

    Chaos Dashboard 会自动创建一个命名为 “entry” 串行节点，作其用为该工作流的入口。

    :::

    ![Choose Task Type](./img/choose-task-type.png)

2. 填写实验信息

    配置方法与创建普通的混沌实验相同。比如，你可以设置一个 POD KILL 类型的 PodChaos 故障，并将其命名为 “kill-nginx”。

    ![Create podkill in Workflow](./img/create-podkill-in-workflow.png)
```

**Comment by STRRL** [comment]
Updated PTAL @en-jin19

File: i18n/zh/docusaurus-plugin-content-docs/version-2.1.2/create-chaos-mesh-workflow.md [hunk 16]
```diff
@@ -24,6 +24,44 @@ Chaos Mesh Workflow 在设计时一定程度上参考了 Argo Workflow。如果
...
+
+### 第 3 步：配置工作流节点
+
+1. 根据需求选择**选择任务类型**
+
+    在本示例中选择的任务类型为”单一“。
+
+    :::note 注意
+
+    Chaos Dashboard 会自动创建一个命名为 “entry” 串行节点，作其用为该工作流的入口。
```
Comment (en-jin19) [inline i18n/zh/docusaurus-plugin-content-docs/version-2.1.2/create-chaos-mesh-workflow.md:47]:
```suggestion
    Chaos Dashboard 会自动创建一个命名为 “entry” 的串行节点，作其用为该工作流的入口。
```

File: i18n/zh/docusaurus-plugin-content-docs/current/create-chaos-mesh-workflow.md [hunk 17]
```diff
@@ -24,6 +24,44 @@ Chaos Mesh Workflow 在设计时一定程度上参考了 Argo Workflow。如果
...
+
+### 第 3 步：配置工作流节点
+
+1. 根据需求选择**选择任务类型**
+
+    在本示例中选择的任务类型为”单一“。
+
+    :::note 注意
+
+    Chaos Dashboard 会自动创建一个命名为 “entry” 串行节点，作其用为该工作流的入口。
```
Comment (en-jin19) [inline i18n/zh/docusaurus-plugin-content-docs/current/create-chaos-mesh-workflow.md:47]:
```suggestion
    Chaos Dashboard 会自动创建一个命名为 "entry" 的串行节点，作其用为该工作流的入口。
```

**Comment by STRRL** [comment]
Updated PTAL @en-jin19

**Review by en-jin19 — APPROVED** [review]
LGTM

**Pull request merged**